Setup/UpgradeData.php

```diff
@@ -234,5 +234,38 @@ public function upgrade(
             $this->resourceConfig->deleteConfig('tax/taxjar/categories', 'default', '');
             $this->eventManager->dispatch('taxjar_salestax_import_categories');
         }
+
+
+        if (version_compare($context->getVersion(), '1.0.5', '<')) {
+            $code = 'tj_ptc';
+            $eavSetup = $this->eavSetupFactory->create(['setup' => $setup]);
+
+            $eavSetup->addAttribute(
+                \Magento\Catalog\Model\Product::ENTITY,
+                $code,
+                [
+                    'group' => 'General',
+                    'type' => 'text',
+                    'label' => 'TaxJar PTC',
```

Comment:
I think consistency with the [tax class dropdown](https://www.taxjar.com/guides/integrations/magento2/#product-sales-tax-exemptions) works here: `TaxJar Category`. Can we display a basic dropdown for product categories similar to the Tax Class view so the merchant knows the PTC category name when reviewing the product?